src/chigame/users/views.py

```diff
@@ -41,3 +42,8 @@ def get_redirect_url(self):
 
 
 user_redirect_view = UserRedirectView.as_view()
+
+def user_list(request):
```

Comment:
should this view only be limited to logged in users?

Reply:
I added in a login requirement for the function on line 47 of views.py